free-courses-pt_BR.md

```diff
@@ -130,6 +131,11 @@
 * [Fast & Furious](https://www.youtube.com/playlist?list=PLy5T05I_eQYOoUz2TtAqq35RLCc-xBZCe) - Codecasts (YouTube)
 
 
+### Vue.js
```

Comment:
Could you please slot this category in alphabetical order in line with the rest of this document.